llmstack/cli.py

```diff
@@ -197,51 +231,69 @@ def signal_handler(sig, frame):
     runner_environment = {}
     with open(env_path) as f:
         config = toml.load(f)
-        for key in config['llmstack']:
-            os.environ[key.upper()] = str(config['llmstack'][key])
-            llmstack_environment[key.upper()] = str(config['llmstack'][key])
-        for key in config['llmstack-runner']:
-            os.environ[f'RUNNER_{key.upper()}'] = str(
-                config['llmstack-runner'][key])
-            runner_environment[f'RUNNER_{key.upper()}'] = str(
-                config['llmstack-runner'][key])
-
-    if len(sys.argv) > 1 and sys.argv[1] == 'runserver':
-        print('Starting LLMStack')
-        run_server_command = ['manage.py',
-                              'runserver', os.environ['LLMSTACK_PORT']]
-        if 'windows' in platform.platform().lower():
-            run_server_command.append('--noreload')
-        run_django_command(
-            run_server_command)
+        for key in config["llmstack"]:
+            os.environ[key.upper()] = str(config["llmstack"][key])
+            llmstack_environment[key.upper()] = str(config["llmstack"][key])
+        for key in config["llmstack-runner"]:
+            os.environ[f"RUNNER_{key.upper()}"] = str(config["llmstack-runner"][key])
+            runner_environment[f"RUNNER_{key.upper()}"] = str(
+                config["llmstack-runner"][key]
+            )
+
+    # Load cli args
+    args = parser.parse_args()
+    if args.port is not None:
+        os.environ["LLMSTACK_PORT"] = args.port
+    if args.host is not None:
+        os.environ["HOST"] = args.host
```

Comment:
Where is this environ variable used? Can you please config if these changes work ?

Reply:
All configs are loaded into env. variables, in line 306 `os.environ["LLMSTACK_PORT"]` is used as port.